Packs/SplunkPy/Integrations/SplunkPy/SplunkPy.py

```diff
@@ -1952,15 +1952,22 @@ def build_search_query(args):
     return query
 
 
-def create_entry_context(args: dict, parsed_search_results, dbot_scores, status_res):
+def create_entry_context(args: dict, parsed_search_results, dbot_scores, status_res, job_id):
     ec = {}
+    number_of_results = len(parsed_search_results)
 
     if args.get('update_context', "true") == "true":
         ec['Splunk.Result'] = parsed_search_results
         if len(dbot_scores) > 0:
             ec['DBotScore'] = dbot_scores
         if status_res:
-            ec['Splunk.JobStatus(val.SID && val.SID === obj.SID)'] = status_res.outputs
+            ec['Splunk.JobStatus(val.SID && val.SID === obj.SID)'] = {
+                **status_res.outputs, 'total_results': number_of_results}
+    if job_id and not status_res:
+        status = 'DONE' if (number_of_results > 0) else 'NO RESULTS'
+        ec['Splunk.JobStatus'] = [{'sid': job_id,
+                                   'total_results': number_of_results,
+                                   'Status': status}]
```

Comment:
Please add the new fields to the yml file. Please change `sid` to `SID` as it currently exists in the yml output.
The name `total_results`  doesn't match the existing standard either. `TotalResults` will be better.
<img width="316" alt="image" src="https://user-images.githubusercontent.com/20818773/213907875-6aadb25f-e0e3-4c03-9989-96b83d326210.png">